src/Domain/Package.php

```diff
@@ -134,7 +134,13 @@ public function is_experimental_build() {
 	 *
 	 * @return boolean
 	 */
-	public function is_feature_plugin_build() {
-		return $this->feature()->is_feature_plugin_build();
-	}
+	// phpcs:disable Squiz.PHP.CommentedOutCode
+	// phpcs:disable Squiz.Commenting.InlineComment.InvalidEndChar
+	// phpcs:disable Squiz.Commenting.InlineComment.SpacingBefore
+	//	public function is_feature_plugin_build() {
+	//		return $this->feature()->is_feature_plugin_build();
+	//	}
+	// phpcs:enable Squiz.PHP.CommentedOutCode
+	// phpcs:enable Squiz.Commenting.InlineComment.InvalidEndChar
+	// phpcs:enable Squiz.Commenting.InlineComment.SpacingBefore
```

Comment:
Any reason we're commenting out those functions?

Reply:
They aren't used and it seems reasonable to keep it around in case we add new blocks that should be behind this feature flag. Though I see a bunch in `FeatureGating.php` that aren't used and also not commented out. Should we keep it around uncommented?